docs/client.rst

```diff
@@ -894,6 +894,27 @@ The request will be sent in the next ``run_session`` function call.
 
 ------
 
+.. code-block:: c
+    bool uxr_prepare_output_stream_fragmented(uxrSession* session, uxrStreamId stream_id, uxrObjectId datawriter_id,
+                                  struct ucdrBuffer* ub, size_t topic_size, uxrOnBuffersFull flush_callback);
+
+Requests a writing into a specific output stream.
+This function will initialize an ``ucdrBuffer`` struct where a topic of ``topic_size`` size must be serialized.
+Whether the necessary gap for writing a ``topic_size`` bytes into the reliable stream, the returned value is ``true``, otherwise ``false``.
+The topic will be sent in the next ``run_session`` function. If buffer is full at some moment during the serialization process, ``flush_callback`` will be call and user will be in charge of running a session for flushing the reliable stream.
+
+NOTE: This approach is not valid with best-effort streams.
+
+:session: Session structure previously initialized.
+:stream_id: The output stream ID where the message will be written.
+:datawriter_id: The `datawriter` ID that will write the topic to the DDS World.
+:mb_topic: An ``ucdrBuffer`` struct used to serialize the topic.
```

Comment:
choose a consistent name for the :mb_topic: (see above).

Reply:
Done